tools/local-env/scripts/utils.js

```diff
@@ -0,0 +1,50 @@
+const local_env_utils = {
+
+	/**
+	 * Determines which Docker compose files are required to properly configure the local environment given the
+	 * specified PHP version, database type, and database version.
+	 */
+	determine_compose_files: function() {
+		process.env.LOCAL_COMPOSE_FILE = '-f docker-compose.yml';
```

Comment:
It feels weird to override env variables that someone might set. I think it would be better to return the compose file string we want to use.

Reply:
I changed `determine_compose_files()` to `get_compose_files()` and updated it to return the files instead.